app/models/incident.rb

```diff
@@ -22,6 +22,8 @@ class Incident
   # TODO: Change the field name from 'ursus_id_str' to 'id_str' or 'incident_id_str'.
   field :ursus_id_str, :string
 
+  global_secondary_index hash_key: :ursus_id_str, projected_attributes: [hash_key]
```

Comment:
Think of it as a view that gets updated whenever the table itself gets updated. Since dynamodb can only query based on a single key, this architecture is necessarily to allow querying on more than one kind of key.